extensions/adobe/experience/content-intelligence-experienceevent.example.2.json

```diff
@@ -0,0 +1,123 @@
+{
+  "_id": "1c796d36-f2e7-4125-813f-7223df5e7b5a-0",
+  "xdm:eventType": "web.webinteraction.linkClicks",
+  "xdm:timestamp": "2020-02-19T16:46:54.963Z",
+  "xdm:identityMap": {
+    "xdm:ECID": [
```

Comment:
these are intended to be a identitiy namespaces. I don't believe `xdm:ECID` is correct.

Reply:
Should be just the Namespace Name. It's a MAP and you are describing the "Key" of the map here. Remove the "xdm:"